ansible/roles/pmm2-images/tasks/main.yml

```diff
@@ -0,0 +1,249 @@
+---
+    - name: Packages                   | Add Nginx repository
+      yum_repository:
+        name: nginx
+        description: nginx repo
+        baseurl: http://nginx.org/packages/mainline/centos/7/$basearch/
+        gpgcheck: no
+        enabled: yes
+
+    - name: Packages                   | Add PMM2 Server YUM repository
+      yum_repository:
+        name: pmm2-laboratory
```

Comment:
I think we should use a different name there that does not match any real repo name. pmm2-server?

Reply:
ok